jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/BufferedContentSink.java

```diff
@@ -35,6 +35,19 @@
  */
 public class BufferedContentSink implements Content.Sink
 {
+    /**
+     * An empty byte array, which if {@link #write(boolean, ByteBuffer, Callback) written}
+     * as a {@link ByteBuffer#wrap(byte[]) wrapped ByteBuffer}
+     * will invoke a {@link #flush(Callback)} operation.
+     */
+    public static final byte[] FLUSH_BYTES = new byte[0];
```

Comment:
Well it is kind of a fallback option in case the BufferedContentSink is wrapped by something that only provides a `byte[]` API.    But then I guess there is no guarantee that it will be wrapped and passed through anyway.   OK